### ISSUE TITLE
TypeError: Object(...) is not a function

### ISSUE THREAD

**Issue by Simoneth** [NoriSte/react-crypto-compare#104]
Hi,
I got this error:
![screenshot-localhost-3000-2019 07 24-19-30-57](https://user-images.githubusercontent.com/8775559/61815417-81341d80-ae4a-11e9-9a7b-ceb88fa5736e.png)

Here is my code (`index.js`):

```
import React, { Component } from "react";
import { connect } from "react-redux";
import { Typography } from "../../includes";
import CryptoCompare from "react-crypto-compare";
// ...

class Dashboard extends Component {
  render() {
    return (
      <Typography variant="display1" gutterBottom component="h2">
        Dashboard
        <CryptoCompare from="EUR" to="BTC" amount={10} apikey="myapi" />
      </Typography>
    );
  }
}
```

**Comment by NoriSte** [comment]
What React version are you using? It seems that the React Hooks are not available in the version you are using!
Let me know, then I'll specify the React version compatibility!

**Comment by Simoneth** [comment]
Here my `package.json`

```
{
  "name": "login-basic",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@material-ui/core": "^3.1.1",
    "@material-ui/icons": "^3.0.1",
    "classnames": "^2.2.6",
    "prop-types": "15.6.0",
    "react": "^16.5.2",
    "react-bootstrap": "0.31.5",
    "react-crypto-compare": "^1.3.0",
    "react-dom": "^16.5.2",
    "react-material-ui-form-validator": "^2.0.2",
    "react-redux": "^5.0.7",
    "react-router-dom": "^4.3.1",
    "react-scripts": "1.1.5",
    "redux": "^4.0.0",
    "redux-logger": "^3.0.6",
    "redux-thunk": "^2.3.0"
  }
}
```

**Comment by NoriSte** [comment]
React Hooks are available from [React 16.8+](https://it.reactjs.org/docs/hooks-intro.html), update it with `npm update react`, update some eventually incompatible packages and let me know if it works 👍
